plugins/modules/keycloak_authentication.py

```diff
@@ -313,6 +311,9 @@ def create_or_update_executions(kc, config, realm='master'):
                             if key not in ("flowAlias", "authenticationConfig", "subFlowType"):
                                 updated_exec[key] = new_exec[key]
                         if new_exec["requirement"] is not None:
+                            # Load the existing execution so we can set the priority (API requires priority since keycloak 25)
+                            execution = kc.get_execution(id_to_update, flow_alias_parent, realm=realm)
+                            updated_exec["priority"] = execution["priority"]
```

Comment:
Is `priority` always present, also for older verisons? If not, you need to check whether it's present first:
```suggestion
                            if "priority" in execution:
                               updated_exec["priority"] = execution["priority"]
```

Reply:
It is present since keycloak 25, but then again older versions are EOL https://endoflife.date/keycloak -- not sure about the policy here…

Reply:
If the module doesn't explicitly says which Keycloak versions it supports, then the PR must work for all Keycloak versions the module worked with before. It doesn't matter to us whether they are EOL or not. The only exception are services that you cannot host locally, there we don't have to care about backwards compatibility...

(We can change the range of supported Keycloak versions in future community.general versions by deprecating support for them, it just cannot be done *now*.)